Applications/Spire/Source/Ui/Button.cpp

```diff
@@ -113,7 +121,7 @@ Button* Spire::make_label_button(const QString& label, QWidget* parent) {
     set(TextColor(QColor::fromRgb(0, 0, 0))).
     set(border(scale_width(1), QColor::fromRgb(0, 0, 0, 0))).
     set(horizontal_padding(scale_width(8)));
-  style.get(Hover() / Body()).
+  style.get((Hover() || Press()) / Body()).
     set(BackgroundColor(QColor::fromRgb(0x4B, 0x23, 0xA0))).
     set(TextColor(QColor::fromRgb(0xFF, 0xFF, 0xFF)));
   style.get(Focus() / Body()).set(
```

Comment:
The disabled styling only changes the text_color, not the background color, remove that from the implementation.